ApollosProject/apollos-apps
# Adds PrayerCard and PrayerInput component

## DESCRIPTION
Currently tests don't pass and I'm not sure how to fix it without removing hooks. Which I could totally do if that was my only option.

<img src="https://user-images.githubusercontent.com/2053547/83076007-c749a700-a042-11ea-86bc-585eaa75f579.gif" width="30%" /><img src="https://user-images.githubusercontent.com/2053547/83076888-5905e400-a044-11ea-8710-06df21272d4e.gif" width="30%" />

### What does this PR do, or why is it needed?
This PR adds `PrayerCard` and `PrayerInput` components. Together these make up the card UI for prayer input and viewing.

This PR adds some extra code for viewing these cards to the `PrayerSwiper` stories as well just to get things on the screen. This story code will become the bases for the `PrayerScreen` component in #1466.

### What design trade-offs/decisions were made?
I intentionally made these components more flexible than I really needed to. It didn't create any more work and it will allow these components to power a wider "prayer" experience including anonymous prayer and personalized prayer prompts.

### How do I test this PR?
Check out the stories! You'll need to run this PR on [this branch](https://github.com/ApollosProject/apollos-templates/pull/51).

### What automated tests did you write?
Should have pretty good coverage.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
